### PR TITLE
Update Front Page Dashboard to include High Level Findings and Vuln Severities

### LINKED ISSUE #172: Design VS Graph(s)
# 💡 Summary #

We need visually appealing graphics to represent the Vulnerability Scan (VS) components on our dashboard. These graphics will enhance the user experience and provide a clear visual representation of these crucial elements.

## Motivation and context ##

A required report metric.

## Implementation notes ##

VS Graphics Design:
- Create a graph(s) or visual that symbolizes each service.
- Ensure the design aligns with our dashboard’s overall aesthetics.
- Maintain consistency with other dashboard graphics.

## Acceptance criteria ##

How do we know when this work is done?

- [x] The graphics are visually appealing and easily distinguishable.
- [x] The designs align with our dashboard’s aesthetics.

### PR DESCRIPTION
**Pull request by ameliav**
#  #

## 🗣 Description ##
Visually appealing graphics to represent  High Level Findings and Vuln Severities components in Crossfeed's front page dashboard

## 💭 Motivation and context ##
Enhance user experience and provide clear visual representation of crucial elements

## 🧪 Testing ##
Updated front page reflects the stakeholders' vision and the metrics coordinate with the org's data

## 📷 Screenshots ##

Dashboard with test data and an org selected
image

Dashboard with test data and no org selected
image

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [ ] Tests have been added and/or modified to cover the changes in this PR.
- [ ] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

File: backend/src/api/app.ts [hunk 1]
```diff
@@ -476,7 +483,7 @@ authenticatedRoute.get('/assessments', handlerToExpress(assessments.list));
 authenticatedRoute.get('/assessments/:id', handlerToExpress(assessments.get));
 
 //************* */
-//  V2 Routes   //
+//  V2 Routes     //
```
Comment (cduhn17) [inline backend/src/api/app.ts]: Did you mean to have an extra space?

Comment (DJensen94) [reply backend/src/api/app.ts]: No we did not: fixed in this commit: [bbf5ab1](https://github.com/cisagov/XFD/pull/237/commits/bbf5ab1e5dafd63daf8443f7bf654f77a6a84fb7)

File: frontend/src/types/index.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ export * from './user';
 export * from './role';
 export * from './scan-task';
 export * from './saved-search';
+export * from './stats';
```
Comment (cduhn17) [inline frontend/src/types/index.ts:12]: Also if there are any locations that we can get away from hard coded values and either parameterize(AWS parameter store) or driven by DB values. In the long run we can avoid major review boards in the event we need to modify values in the future.

**Review by cduhn17 — COMMENTED** [review]
See a minor comments.

File: frontend/src/types/index.ts (hunk 2 above)
Comment (cduhn17) [reply frontend/src/types/index.ts:12]: @DJensen94 
As mentioned this is more of an idea as an attempt at cutting down on the big review processes we have to deal with.

**Review by cduhn17 — APPROVED** [review]
LGTM

**Review by schmelz21 — APPROVED** [review]

**Pull request merged**